modules/@ergonode/categories/src/extends/components/Forms/ProductFormCategory.vue

```diff
@@ -45,12 +68,31 @@ export default {
         categoryFieldKey() {
             return 'categories';
         },
+        placeholder() {
+            return {
+                title: 'No categories',
+                subtitle: 'There are no categories in the system, so you can create the first one.',
+            };
+        },
+        isAllowedToCreate() {
+            return this.$hasAccess([
+                PRIVILEGES.CATEGORY.create,
+            ]);
+        },
+        smallSize() {
+            return SIZE.SMALL;
+        },
     },
     methods: {
         ...mapActions('product', [
             '__setState',
         ]),
-        setCategoriesValue(value) {
+        onNavigateToCategories() {
+            this.$router.push({
+                name: 'categories-grid',
```

Comment:
I would create a configuration file and import that as constants throughout the app 
also in router 

`export const CATEGORIES_GRID = 'categories-grid';

export default {
    CATEGORIES_GRID
};


this.$router.push({
                name: CATEGORIES_GRID,
})
`

Reply:
That's good idea. We will create separate ticket for this, cuz it should be implemented across whole app.